packages/indexing/src/database-index-api.ts

```diff
@@ -234,7 +234,20 @@ export abstract class DatabaseIndexApi<DateType = Date | number> {
     }
     const toMerge = cloneDeep(indexedData)
     delete toMerge['created_at']
-    await this.dbConnection(tableName).insert(indexedData).onConflict('stream_id').merge(toMerge)
+    await this.dbConnection(tableName)
+      .insert(indexedData)
+      .onConflict('stream_id')
+      .merge(toMerge)
+      .catch((err) => {
+        this.logger.err(
+          `Error indexing ${JSON.stringify(
```

Comment:
if you're going to log json data, I suggest putting it at the very end of the log message, otherwise the message can get hard to read.  So I'd suggest rewording slightly so that model name and the error message comes first, with the arbitrary json data at the end.